superset/base/params.env

```diff
@@ -16,4 +16,4 @@ SUPERSET_DB_MEMORY_REQUESTS=300Mi
 SUPERSET_DB_MEMORY_LIMITS=1Gi
 SUPERSET_DB_CPU_REQUESTS=300m
 SUPERSET_DB_CPU_LIMITS=1
-SUPERSET_IMAGE=quay.io/internaldatahub/superset:1.1.0
+SUPERSET_IMAGE=quay.io/internaldatahub/superset:1.1.0-trino
```

Comment:
Should this image reside in the opendatahub org on Quay?

Reply:
@maulikjs @LaVLaS

Reply:
`opendatahub/superset` is empty, so I decided to keep the version tag without the additional name: 

```
quay.io/opendatahub/superset:1.1.0
```

@4n4nd change the code to point to this image

Reply:
does it have the trino driver?

Reply:
Yes, I pulled from the `1.1.0-trino` in internaldatahub org to there. I just didn't use the `-trino` suffix.

Reply:
updated :+1: